### PR TITLE
Link for formiz v2

### PR DESCRIPTION
**Pull request by doriandel**
added a small link to the issue that announce formiz v2 in the hero of formiz landing page, configured the link's style and hover effect.
image

### CODE REVIEW

File: documentation/src/pages/index.js [hunk 1]
```diff
@@ -170,6 +170,15 @@ function Home() {
...
           </div>
+          <Link
+            className={classnames(
+              'button button--link button--md text-white underline transition duration-700 ease-in-out hover:no-underline hover:text-green-100',
+              styles.getStarted,
+            )}
+            to={useBaseUrl('https://github.com/ivan-dalmet/formiz/issues/116')}
+          >
+            Formiz v2 is under construction !
+          </Link>
```
Comment (yoannfleurydev) [inline documentation/src/pages/index.js:181]: 1. I think that you can use a basic HTML `a` tag for this as `Link` is [made to navigate in Docusaurus](https://docusaurus.io/docs/docusaurus-core#link) and not outside.
2. I'm not sure about `useBaseUrl` as the name doesn't match what we want to do here.
3. Be careful to use `target="_blank"` to open the link in a new tab and be careful about its security.
4.. Thanks for the contribution.

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: documentation/src/pages/index.js (hunk 1 above)
Comment (doriandel) [reply documentation/src/pages/index.js:181]: Thank you for your review ! Currently working on fixing what I've done.

**Comment by doriandel** [comment]
Did some fixes following @[yoannfleurydev](https://github.com/yoannfleurydev) review : 

1. changed the link from a `Link` component to a basic HTML `a` tag
2. Removed `useBaseUrl`
3. added `target="_blank"` but with `rel="noreferrer"` for security purpose

File: documentation/src/pages/index.js [hunk 2]
```diff
@@ -170,6 +170,17 @@ function Home() {
...
+          <a
+            className={classnames(
+              'button button--link button--md text-white underline transition duration-700 ease-in-out hover:no-underline hover:text-green-100',
+              styles.getStarted,
+            )}
+            href="https://github.com/ivan-dalmet/formiz/issues/116"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Formiz v2 is under construction !
```
Comment (ivan-dalmet) [inline documentation/src/pages/index.js:182]:
```suggestion
            Formiz v2 is under construction!
```

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]

File: documentation/src/pages/index.js (hunk 2 above)
Comment (doriandel) [reply documentation/src/pages/index.js]: Fixed ! Thank you for your review !

**Pull request merged**